dandi/organize.py

```diff
@@ -307,9 +308,28 @@ def _assign_obj_id(metadata, non_unique):
     # Avoid heavy import by importing within function:
     from .pynwb_utils import get_object_id
 
-    msg = "%d out of %d paths are not unique" % (len(non_unique), len(metadata))
+    msg = "%d out of %d paths are not unique." % (len(non_unique), len(metadata))
+    non_unique_paths = sorted(non_unique)
 
-    lgr.info(msg + ". We will try adding _obj- based on crc32 of object_id")
+    # provide more information to the user
+    def get_msg(path, indent="  "):
+        in_paths = non_unique[path]
+        return (
+            f"{len(in_paths)} paths 'compete' for the path {path!r}:"
+            + f"\n{indent}".join([""] + in_paths)
+        )
+
+    msg += "\n " + get_msg(non_unique_paths[0])
+    if len(non_unique) > 1:
+        if not lgr.isEnabledFor(logging.DEBUG):
```

Comment:
@jwodder 
this condition is not effective -- we seems always enabled and effectiveLevel is 0.  I guess due to having multiple backends etc. Is there a way to make it dependent on what user specifies in cli?

Reply:
I don't think this condition is a good idea in the first place; the content of log messages should not depend on whether another log messages would be shown.  If you really want to display some paths at INFO and the full paths at DEBUG, then always log two messages.